modular_skyrat/modules/goofsec/jobs/sol_fed.dm

```diff
@@ -178,28 +262,37 @@ GLOBAL_LIST_INIT(emergency_responders, list())
 	if(!GLOB.amt_911_responders)
 		to_chat(user, "There's no 911 responders. Stop spawning shit that you don't know how to use.")
 		return
+	if(!user.mind.has_antag_datum(/datum/antagonist/ert/request_911))
+		to_chat(user, "You are not a 911 responder. You cannot vote for a false call.")
+		return
+
 	if(!activated && !GLOB.fradulent_911_declared)
+		if(tgui_input_list(user, "Did the station make a fraudulent 911 call?", "Fraudulent 911 Call Reporter", list("Yes", "No")) != "Yes")
+			to_chat(user, "You decide not to declare the station as having made a false 911 call.")
+			return
+		message_admins("[ADMIN_LOOKUPFLW(user)] has voted that the station made a fraudulent 911 call.")
 		activated = TRUE
 		icon_state = "reporter_on"
 		GLOB.votes_for_bad_911_call++
 		to_chat(user, span_warning("You have reported the 911 call as fradulent. \
 		Current Votes: [GLOB.votes_for_bad_911_call]/[GLOB.amt_911_responders] votes."))
-		if(GLOB.votes_for_bad_911_call >= (GLOB.amt_911_responders / 2))
+		if(GLOB.votes_for_bad_911_call >= GLOB.amt_911_responders)
 			to_chat(user, span_warning("The 911 call has been declared fradulent!"))
 			GLOB.fradulent_911_declared = TRUE
 			var/datum/bank_account/station_balance = SSeconomy.get_dep_account(ACCOUNT_CAR)
 			station_balance?._adjust_money(-20000) // paying for the gas to drive all the fuckin' way out to the frontier
```

Comment:
What happens if the station doesn't have the funds?

Reply:
Caps at 0 credits.